commit 09cdb7083193e483030f3ceb376923745302160f
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Wed Jul 19 14:50:49 2017 +0100

    Revert "Revert "bcm2835-mmc: Fix DMA usage""
    
    This reverts commit d52c1ae3a211386b5ec98804a468436e85b5b8af.

diff --git a/drivers/mmc/host/bcm2835-mmc.c b/drivers/mmc/host/bcm2835-mmc.c
index 031ec56eee1a..c4a5e992c6fb 100644
--- a/drivers/mmc/host/bcm2835-mmc.c
+++ b/drivers/mmc/host/bcm2835-mmc.c
@@ -1364,14 +1364,14 @@ static int bcm2835_mmc_add_host(struct bcm2835_host *host)
 		if (ret == 0) {
 			host->dma_cfg_rx = cfg;
 
-			host->use_dma = true;
+			host->have_dma = true;
 		} else {
 			pr_err("%s: unable to configure DMA channel. "
-			       "Faling back to PIO\n",
+			       "Falling back to PIO\n",
 			       mmc_hostname(mmc));
 			dma_release_channel(host->dma_chan_rxtx);
 			host->dma_chan_rxtx = NULL;
-			host->use_dma = false;
+			host->have_dma = false;
 		}
 	}
 #endif
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1234_09cdb7083193e483030f3ceb376923745302160f.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1234_09cdb7083193e483030f3ceb376923745302160f.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
